plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardProjectConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+/**
+ * Collects all data needed to create and configure an App Engine Standard Project.
+ */
+class AppEngineStandardProjectConfig {
+
+  private String appEngineProjectId;
+  private String eclipseProjectName;
+  private String packageName;
+  private String eclipseProjectDirectory;
+
+  // todo does builder pattern make more sense here?
```

Comment:
Sorry, I wasn't clear.  What I meant was: We'll have different New wizards for AE Flex, GKE, and GCE projects, and we'll want to share as many pieces between them.  So one approach would have them all share this "Create a Google Developers Project" page (configure Eclipse project name and location, and GDP Id), and then have some additional pages for specifics of AE Standard, AE Flex, GKE, GCE.  Minimize churn.